### PR TITLE
Retire: Add example alerts

### PR DESCRIPTION
**Pull request by psiinon**
Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
PR for the generated alert: https://github.com/zaproxy/zaproxy-website/pull/233

**Review by kingthorin — APPROVED** [review]

File: addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java [hunk 1]
```diff
@@ -88,27 +91,39 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
...
+    }
+
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<Alert>();
+        alerts.add(
+                buildAlert(
+                                new Result(
+                                        "ExampleLibrary",
+                                        "x.y.z",
+                                        Collections.<String, Set<String>>emptyMap(),
```
Comment (thc202) [inline addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java:120]: The types could be removed.

Comment (psiinon) [reply addOns/retire/src/main/java/org/zaproxy/addon/retire/RetireScanRule.java]: Done :)

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thanks!

**Review by kingthorin — APPROVED** [review]

**Pull request merged**